lib/howl/application.moon

```diff
@@ -316,12 +324,29 @@ class Application extends PropertyObject
 
       howl.janitor.start!
 
+    -- load files from command line
+    loaded_buffers = {}
     for path in *files
       file = File path
-      buffer = @new_buffer mode.for_file file
-      buffer.file = file
-      signal.emit 'file-opened', :file, :buffer
+      buffer = @_buffer_for_file file
+      unless buffer
+        buffer = @new_buffer mode.for_file file
+        buffer.file = file
+        signal.emit 'file-opened', :file, :buffer
+
+      append loaded_buffers, buffer
+
+    -- files we've loaded via a --reuse invocation should be shown
+    if #loaded_buffers > 0 and @_loaded
+      for i = 1, math.min(#@editors, #loaded_buffers)
+        @editors[i].buffer = loaded_buffers[i]
+
+    -- all loaded files should be considered as having been viewed just now
+    now = howl.sys.time!
+    for b in *loaded_buffers
+      b.last_shown = now
```

Comment:
We need it since we just load buffers from the command line straight into buffers without passing any editor. For the --reuse case we then explicitly fill any already open editors.